### DISCUSSION TITLE
bug whent i start a game with mc_faction

### DISCUSSION THREAD

**Discussion by erb421** [minetest-palamod/mc_faction#11] [General]
/home/eliott/.minetest/mods/mc_faction-master/init.lua:
cannot open /home/eliott/.minetest/mods/mc_faction-master/1: Aucun fichier ou dossier de ce type
stack traceback:
	[C]: in function 'dofile'
	/home/eliott/.minetest/mods/mc_faction-master/init.lua:11: in main chunk
Voir « debug.txt » pour plus d'informations.

**Comment by AFCMS** [comment]
@erb421 I just checked and effectively it doesn't run on latest Minetest, but the mod is in a non-functional state at the moment anyways.
